pkg/exchange/ftx/exchange.go

```diff
@@ -33,6 +34,7 @@ func NewExchange(key, secret string, subAccount string) *Exchange {
 	if err != nil {
 		panic(err)
 	}
+	ftxExchangeSymbolMap = make(map[string]string)
```

Comment:
how about putting this in the init function?

Reply:
I did not found the init function of ftx exchange, do you have any recommendation?

Reply:
you can add one?